### PR TITLE
Form builder UI docs update

### PR DESCRIPTION
**Pull request by Bas02**
Yo

Form builder docs update with new UI. 
Pushed as Draft first to see changes, still need to add last parts, waiting for deploy currently.. =)

### CODE REVIEW

File: docs/Forms.md [hunk 1]
```diff
@@ -1,141 +1,69 @@
...
+For this example, click the Address form. 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder2.png)
+
+We can now drag around the fields pre-made fields by BTCPay Server. 
+You can re-order them or create a new field by clicking on the `Add form Element` at the bottom of the form. 
+
+## Create a custom form.
+
+You might have a different use case, a restaurant, and you need to know the table number, how to call the customer when serving them, allergies, and special requests.
```
Comment (webwworthy) [inline docs/Forms.md:26]:
```suggestion
You might have a different use case. For example if you own a restaurant you need to know the table number, how to call the customer when serving them, allergies, and special requests. 
```

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -71,17 +71,19 @@ const sidebarUserGuide = [
...
       {
         title: 'Pull Payments',
         path: '/PullPayments',
         children: [['/Refund', 'Refunds']]
       },
       ['/Payouts', 'Payouts'],
-      ['/Forms', 'Forms'],
+      {
+        title: 'Form Builder',
+        path: '/forms',
```
Comment (dennisreimann) [inline docs/.vuepress/config.js:84]: Let's stick to the camelcase convention here:

```suggestion
        path: '/Forms',
```

File: docs/Forms.md [hunk 3]
```diff
@@ -1,141 +1,69 @@
...
-Here are some use cases for this feature:
-
-- `Assisting user input`: Pre-fill fields with known customer information to make it easier for them to complete the form. For example, if you already know a customer's email address, you can pre-fill the email field to save them time.
-- `Personalization`: Customize the form based on customer preferences or segmentation. For instance, if you have different customer tiers, you can pre-fill the form with relevant data, such as their membership level or specific offers.
-- `Tracking`: Track the source of customer visits by using hidden fields and pre-filled values. For example, you can create different links with pre-filled `utm_media` values for each marketing channel (e.g., Twitter, Facebook, email). This helps you analyze the effectiveness of your marketing efforts.
-
-- `A/B testing`: Pre-fill fields with different values to test different versions of a form, enabling you to optimize the user experience and conversion rates.
+BTCPay Server's Forms Builder enables you to request specific information from your customer. 
+
+These forms are fully customizable to suit your requirements. By dragging and dropping the fields you want!
```
Comment (dennisreimann) [inline docs/Forms.md:5]: This sentence seems weird at that place imho:

```suggestion
These forms are fully customizable to suit your requirements.
```

File: docs/Forms.md [hunk 4]
```diff
@@ -1,141 +1,69 @@
...
+In this introduction, we will go through the visual form builder; if you'd like to do more advanced settings, please visit the [Advanced forms](./AdvancedForms.md) section of our documentation.
```
Comment (dennisreimann) [inline docs/Forms.md:6]:
```suggestion
In this introduction, we will go through the visual form builder; if you'd like to do more advanced settings, please visit the [Advanced Forms](./AdvancedForms.md) section of our documentation. 
```

File: docs/Forms.md [hunk 5]
```diff
@@ -1,141 +1,69 @@
...
-- `A/B testing`: Pre-fill fields with different values to test different versions of a form, enabling you to optimize the user experience and conversion rates.
+BTCPay Server's Forms Builder enables you to request specific information from your customer. 
+
+These forms are fully customizable to suit your requirements. By dragging and dropping the fields you want!
+In this introduction, we will go through the visual form builder; if you'd like to do more advanced settings, please visit the [Advanced forms](./AdvancedForms.md) section of our documentation. 
+
+## Setting up your store's first Custom form. 
+
+In this example, we will start with creating a standard form we've pre-made. 
+Click on Store settings and the last tab of your store settings is Forms, click Forms to create your first Custom Form.
```
Comment (dennisreimann) [inline docs/Forms.md:11]:
```suggestion
Click on Store Settings and the last tab of your store settings is Forms. Click Forms to create your first Custom Form. 
```

**Review by dennisreimann — COMMENTED** [review]
The second and third screenshot for the Forms doc could be better when taking them with a not so wide browser window. The details get lost in the preview, because the text is so small.

File: docs/Forms.md [hunk 6]
```diff
@@ -1,141 +1,69 @@
...
+Unlike before, we will start with the empty field that was generated. 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder2-1.png)
+
+1. We will name the first field, `Table number` 
+2. Define the `Type` of the field; we need it to be Text or Number, click on the drop-down and select `Number.`
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder2-2.png)
+
+3. The label we set for this field as it shows to the customer; in our example, we will name it `Table Number.`
```
Comment (petzsch) [inline docs/Forms.md:42]:
```suggestion
3. The label we set for this field as it shows to the customer; in our example, we will name it `Table Number`.
```

File: docs/Forms.md [hunk 7]
```diff
@@ -1,141 +1,69 @@
...
+4. Regarding The name of the field, we replicate the previous field's name, `Table Number,` for consistency.
```
Comment (petzsch) [inline docs/Forms.md:43]:
```suggestion
4. Regarding The name of the field, we replicate the previous field's name, `Table Number`, for consistency. 
```

File: docs/Forms.md [hunk 8]
```diff
@@ -1,141 +1,69 @@
...
+
+## Create a custom form.
+
+You might have a different use case. For example if you own a restaurant you need to know the table number, how to call the customer when serving them, allergies, and special requests. 
+
+Let's create a custom form in the following steps. 
+We will start at the same settings tab from the previous example, Store settings -> Forms.
+
+Click `create form` in the top right. 
+We'll start by giving it a name; in the example, we will use `Restaurant.`
```
Comment (petzsch) [inline docs/Forms.md:32]:
```suggestion
We'll start by giving it a name; in the example, we will use `Restaurant`. 
```

File: docs/Forms.md [hunk 9]
```diff
@@ -1,141 +1,69 @@
...
+We will start at the same settings tab from the previous example, Store settings -> Forms.
+
+Click `create form` in the top right. 
+We'll start by giving it a name; in the example, we will use `Restaurant.` 
+Unlike before, we will start with the empty field that was generated. 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder2-1.png)
+
+1. We will name the first field, `Table number` 
+2. Define the `Type` of the field; we need it to be Text or Number, click on the drop-down and select `Number.`
```
Comment (petzsch) [inline docs/Forms.md:38]:
```suggestion
2. Define the `Type` of the field; we need it to be Text or Number, click on the drop-down and select `Number`.
```

File: docs/Forms.md [hunk 10]
```diff
@@ -1,141 +1,69 @@
...
+
+After you've filled in the parameters for the field, it should show on the left of your editor how the field displays and functions when the customer interacts with it. 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder2-4.png)
+
+Now that the first field is done, you can click on the `+ Add form element` below your first field and create the rest of the required form fields. Once you've made all the fields, click Save in the top right of your screen, and all should be set! 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder3.png)
+
+The `Form Builder` makes creating custom forms flexible and easy to use. If you still require more personalization, as mentioned at the start of this guide, please read on [Advanced Forms](./AdvancedForms.md) to learn about the JSON created in the `Code` tab in the Form builder.
```
Comment (petzsch) [inline docs/Forms.md:58]:
```suggestion
The `Form Builder` makes creating custom forms flexible and easy. If you still require more personalization, as mentioned at the start of this guide, please read on [Advanced Forms](./AdvancedForms.md) to learn about the JSON created in the `Code` tab in the Form builder. 
```

File: docs/Forms.md [hunk 11]
```diff
@@ -1,141 +1,69 @@
...
+
+Now that the first field is done, you can click on the `+ Add form element` below your first field and create the rest of the required form fields. Once you've made all the fields, click Save in the top right of your screen, and all should be set! 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder3.png)
+
+The `Form Builder` makes creating custom forms flexible and easy to use. If you still require more personalization, as mentioned at the start of this guide, please read on [Advanced Forms](./AdvancedForms.md) to learn about the JSON created in the `Code` tab in the Form builder. 
+
+## Support
+
+If you have trouble using BTCPay Server, consider joining the [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from BTCPay community members.
```
Comment (petzsch) [inline docs/Forms.md:62]:
```suggestion
If you have trouble using BTCPay Server, consider joining the [BTCPayServer Mattermost](https://chat.btcpayserver.org) to get help from BTCPay community members.
```

File: docs/Forms.md [hunk 12]
```diff
@@ -1,141 +1,69 @@
...
+
+Now that the first field is done, you can click on the `+ Add form element` below your first field and create the rest of the required form fields. Once you've made all the fields, click `Save` in the top right of your screen, and all should be set! 
+
+![BTCPay Server formbuilder - Create new form](./img/formbuilder/btcpayformbuilder3.png)
+
+The `Form Builder` makes creating custom forms flexible and easy to use. If you still require more personalization, as mentioned at the start of this guide, please read on [Advanced Forms](./AdvancedForms.md) to learn about the JSON created in the `Code` tab in the Form builder. 
+
+## Support
+
+If you have trouble using BTCPay Server, consider joining the [BTCPayServer Mattermost](https://chat.btcpayserver.org) to get help from BTCPay community members.
```
Comment (dennisreimann) [inline docs/Forms.md:62]:
```suggestion
If you have trouble using BTCPay Server, consider joining the [BTCPay Server Mattermost](https://chat.btcpayserver.org) to get help from BTCPay community members.
```

File: docs/Forms.md [hunk 13]
```diff
@@ -1,141 +1,69 @@
...
+
+Only file a [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other community members.
+
+Please check out our [official website](https://btcpayserver.org/),and [FAQ](./FAQ/README.md#btcpay-frequently-asked-questions-and-common-issues) for more details.
```
Comment (dennisreimann) [inline docs/Forms.md:66]:
```suggestion
Please check out our [official website](https://btcpayserver.org/) and [FAQ](./FAQ/README.md#btcpay-frequently-asked-questions-and-common-issues) for more details.
```

**Pull request merged**